pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -536,6 +556,22 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
                 <option value="prometheus">Prometheus</option>
                 <option value="thanos">Thanos</option>
               </Input>
+              <Label style={{ marginLeft: '10px', display: 'inline-block' }} className="control-label">
+                Tenant
+              </Label>
+              <Input
+                style={{
+                  width: 'auto',
+                  marginLeft: '10px',
+                  display: 'inline-block',
+                }}
+                id={`tenant=${id}`}
+                type="text"
+                bsSize="sm"
+                onChange={this.handleChangeTenant}
+                placeholder={`${defaultTenant}`}
+                value={options.tenant}
```

Comment:
Do we need any validation for the tenant value? Or we want to rely on Thanos backend to do it

Reply:
There's currently just the following validation in the backend: https://github.com/thanos-io/thanos/blob/ec6b8b63d64f9570a56bb633f87245e152e5558c/pkg/tenancy/tenancy.go#L38 that seems fine to just do in the backend imo.

I'm not sure if it makes sense to do any more format validation of the tenant? For receive this is also the only thing we have in place.